OCKSample/Main/Profile/ProfileViewModel.swift

```diff
@@ -69,8 +69,13 @@ class ProfileViewModel: ObservableObject {
         }
 
         if patientHasBeenUpdated {
-            _ = try await AppDelegateKey.defaultValue?.store.updatePatient(patientToUpdate)
-            Logger.profile.info("Successfully updated patient")
+            if let anyPatient = try await AppDelegateKey.defaultValue?.store.updateAnyPatient(patientToUpdate),
```

Comment:
Should something similar be done when OCKContact is updated? If so, can you add it to this PR?

Reply:
NVM, I'm thinking about the class implementation...